client/js/config/models.js

```diff
@@ -1,8 +1,17 @@
 import AsyncStorage from '@react-native-community/async-storage';
 
-export const removeOnbording = async () => {
+export const getOnboarding = async () => {
   try {
-    return await AsyncStorage.setItem(`isOnBoardingCompleted`, true);
+    return await AsyncStorage.getItem('isOnBoardingCompleted');
+  } catch (e) {
+    console.log(e);
+    return false;
+  }
+};
+
+export const removeOnboarding = async () => {
```

Comment:
I think "setOnboarding" is a more suitable name here

Reply:
OK